docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -133,3 +133,10 @@ Ansible automatically loads all plugins from certain directories adjacent to you
 .. warning::
 
    Roles contained in collections cannot contain any plugins. All plugins in a collection must live in the collection ``plugins`` directory tree. All plugins in that tree are accessible to all roles in the collection. If you are developing new plugins, we recommend distributing them in :ref:`collections <developing_collections>`, not in roles.
+
+.. _ansible.legacy.custom:
+
+Using ``ansible.legacy`` to access custom versions of an ``ansible.builtin`` module
+===================================================================================
+
+If you need to override one of the ``ansible.builtin` modules, you need to use ``ansible.legacy`` as part of the fully-qualified collection name (FQCN). For example, if you had your own ``copy`` module, you would access it as ``ansible.legacy.copy``. See :ref:`using_ansible_legacy` for details on how to use custom modules with roles within a collection.
```

Comment:
need to override AND are using FQCN (there should be no need if using short names)